tests/ignite/metrics/test_frequency.py

```diff
@@ -57,3 +57,45 @@ def test_frequency_with_engine():
 def test_frequency_with_engine_distributed(distributed_context_single_node_gloo):
     device = "cpu"
     _test_frequency_with_engine(device, workers=dist.get_world_size())
+
+
+def _test_frequency_with_engine_with_every(device, workers, every):
+
+    artificial_time = 0.1 / workers  # seconds
+    total_tokens = 1200 // workers
+    batch_size = 128 // workers
+
+    estimated_wps = batch_size * workers / artificial_time
+
+    def update_fn(engine, batch):
+        time.sleep(artificial_time)
+        return {"ntokens": len(batch)}
+
+    engine = Engine(update_fn)
+    wps_metric = Frequency(output_transform=lambda x: x["ntokens"], device=device)
+    wps_metric.attach(engine, "wps")
```

Comment:
Shouldn't this be `wps_metric.attach(engine, "wps", event_name=Events.ITERATION_COMPLETED(every=every))` ?